src/main/java/com/pinterest/secor/common/SecorSchemaRegistryClient.java

```diff
@@ -68,7 +68,6 @@ public GenericRecord deserialize(String topic, byte[] message) {
         GenericRecord record = (GenericRecord) deserializer.deserialize(topic, message);
         if (record != null) {
             Schema schema = record.getSchema();
-            schemas.put(topic, schema);
```

Comment:
hi @dorjeetsering, just curious here, do we need remove this line, looks like noting get updated to the **record**

Reply:
That's the main change we want. We can remove the null check block.
if (record != null) {
            Schema schema = record.getSchema();
        }

Reply:
thanks @dorjeetsering